Dixit_Logic/Classes/DixitGame.cs

```diff
@@ -198,7 +220,7 @@ public void EvaluatePoints()
                     
 
                     //This section cheks whether all players has figured out the original card
-                    if (goodGuess != (_actGameState.Players.Count - 1))
+                    if (goodGuess < (_actGameState.Players.Count - 1))
                     {                        
                         _actGameState.Points[_actGameState.ActualPlayer] = _actGameState.Points[_actGameState.ActualPlayer] + 3;
                     }
```

Comment:
ahh yes, sorry. But the player gets 3 points even if nobody found out the original card, right? In this case the player should get no points because the hint was too far from the actual card.